README.md

```diff
@@ -97,6 +97,8 @@ juvix interactive
 
 [Ormolu](https://github.com/cryptiumlabs/ormolu) required for source formatting.
 
+[Quicklisp](https://www.quicklisp.org/beta/) and [sbcl](http://www.sbcl.org/) required for generation of code layout documentation. 
```

Comment:
I think something along the lines of what is below would be a bit better.

"required for the automatic generation of documentation in doc/Code" with a link to the folder

Reply:
Done.